omnigibson/prims/joint_prim.py

```diff
@@ -137,6 +138,7 @@ def _initialize(self):
 
         # Update the joint indices etc.
         self.update_handles()
+        self._body_name = self.prim_path.split("/")[-1]
```

Comment:
we already have joint_name, which is probably more appropriate?

Reply:
done